src/aics-image-viewer/components/AxisClipSliders/index.tsx

```diff
@@ -93,7 +94,7 @@ export default class AxisClipSliders extends React.Component<AxisClipSlidersProp
     const numSlices = this.props.numSlices[axis];
     const clipVals = this.props.region[axis];
     const sliderVals = [Math.round(clipVals[0] * numSlices), Math.round(clipVals[1] * numSlices)];
-    const range = { min: 0, max: numSlices - 1 };
+    const range = { min: 0, max: numSlices - (twoD ? 1 : 0) };
```

Comment:
is this a behavior change?

Reply:
i think i see; it is compensated for, below

Reply:
This did end up becoming a subtle behavior change, which I think boils down to the borders of the right sliders' range being increased by one. Before, both sliders went from 0 to n-1 and could touch; now the sliders go from 0 to n and must be at least one step apart. This way you can still grab and move the range when only a single slice is visible. (If I recall correctly, being able to move a single slice like that emphasized the need for a solution to #123 - the combination of two slices was extra visible in 3d where the slice could be viewed edge-on.)